Tracking/python/tracking.py

```diff
@@ -231,10 +231,13 @@ def __init__(self, instance_name='GSFProcessor'):
         self.abortOnError    = False
         self.disableAllMaterialHandling = False
         self.weightCutoff    = 1.0e-4
+        self.debug = True
```

Comment:
```suggestion
        self.debug = False
```
should we have the default on False?